lib/seek/samples/attribute_type_handlers/cv_list_attribute_type_handler.rb

```diff
@@ -0,0 +1,29 @@
+module Seek
+  module Samples
+    module AttributeTypeHandlers
+      class CVListAttributeTypeHandler < BaseAttributeHandler
+        class MissingControlledVocabularyException < AttributeHandlerException; end
+        class NotArrayException < AttributeHandlerException; end
+        class InvalidControlledVocabularyException < AttributeHandlerException; end
+
+        def test_value(array_value)
+
+          if !array_value.is_a?(Array)
+            fail NotArrayException.new unless array_value.is_a?(Array)
+          else
+            array_value.each do |value|
+              fail InvalidControlledVocabularyException.new(value) unless controlled_vocab.includes_term?(value) || controlled_vocab.custom_input
```

Comment:
if you provide a term that isn't in the controlled vocab, won't this result in an unhandled exception rather than validation being false?
There exceptions are passed forward as something unexpected happening, rather than just a validation error - https://github.com/seek4science/seek/blob/master/lib/seek/samples/attribute_type_handlers/base_attribute_handler.rb#L23